static/js/redux/reducers/user_info_reducer.jsx

```diff
@@ -51,6 +52,8 @@ const userInfo = (state = initialState, action) => {
       return Object.assign({}, state, { isVisible: true });
     case ActionTypes.SET_SETTINGS_MODAL_HIDDEN:
       return Object.assign({}, state, { isVisible: false });
+    case ActionTypes.DELETED_ACCOUNT:
```

Comment:
should we set data to `{ isLoggedIn: false}` here/do something to remove user info from the redux state? or does that happen automatically when you navigate to `user/logout`?

Reply:
when you navigate to `user/logout`, the entire redux state is cleared